client/src/worldgen.rs

```diff
@@ -155,10 +155,82 @@ pub fn voxels(graph: &mut DualGraph, node: NodeId, cube: Vertex) -> VoxelData {
                 }
             }
         }
+
+        // Planting trees on dirt. Trees consist of a block of wood and a block of leaves.
+        // The leaf block is on the opposite face of the wood block as the dirt block.
+        let loc: usize = 2;
+        let voxel_of_interest_index =
+            index(absolute_subchunk_coords(loc, loc, loc, subchunk_offset));
+        let neighbor_data = voxel_neighbors(na::Vector3::repeat(loc), subchunk_offset, &mut voxels);
```

Comment:
I'm not sure why this hardcoded location produces good results, but empirically it does. Nice!